<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WebHama::MainWindow</class>
 <widget class="QMainWindow" name="WebHama::MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1344</width>
    <height>820</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>webhama</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QToolButton" name="m_back">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../res/webhama.qrc">
          <normaloff>:/icon/icon/ic_arrow_back_black_24px.svg</normaloff>:/icon/icon/ic_arrow_back_black_24px.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="m_location"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="m_tabs">
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>m_tabs</zorder>
   <zorder></zorder>
  </widget>
  <widget class="QStatusBar" name="m_statusbar"/>
 </widget>
 <resources>
  <include location="../res/webhama.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>OnLoad(bool)</slot>
 </slots>
</ui>
